docs/source/configuration/entity-caching.mdx

```diff
@@ -129,9 +128,162 @@ This entry contains an object with the `all` field to affect all subgraph reques
       "data": "key2"
     }
 }
+```
+
+### Entity cache invalidation
+
+When existing cache entries need to be replaced, the router supports a couple of ways for you to invalidate entity cache entries:
+- **Invalidation endpoint** - the router exposes an invalidation endpoint that can receive invalidation requests from any authorized service out-of-band. A subgraph can use this endpoint to asynchronously trigger invalidation events, or a platform operator can use it to maintain the caching levels.
```

Comment:
In what sense is it asynchronous? The invalidation endpoint waits for the DEL operations to complete and returns a count, which is the number of scanned entries, to the client. That sounds synchronous to me (at least with respect to the client request).

Reply:
asynchronous as in it does not have to do that in the middle of handling a request. That can be sent at any point in time

Reply:
Ah. I find it a bit confusing because the invalidation endpoint itself is synchronous. I'll see if I can think of a better way to describe that.